compose_extrinsic needs additional dependencies

I'm trying to use this lib within my own crate (example with generic extrinsic). The following occurs:

```

error[E0432]: unresolved import `codec`
  --> src/lib.rs:38:14
   |
38 |       let xt = compose_extrinsic!(
   |  ______________^
39 | |         api.clone(),
40 | |         "Balances",
41 | |         "transfer",
42 | |         GenericAddress::from(to),
43 | |         Compact(42 as u128)
44 | |     );
   | |_____^ use of undeclared type or module `codec`
   |
   = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

...
error[E0433]: failed to resolve: use of undeclared type or module `primitives`
...
error[E0432]: unresolved import `log`
```

all of which probably caused by these imports:
https://github.com/scs/substrate-api-client/blob/9e33717bcc2546806ce7201f4772aaa9da3e58f8/src/extrinsic/mod.rs#L68

--- a/src/extrinsic/mod.rs
+++ b/src/extrinsic/mod.rs
@@ -23,6 +23,9 @@ pub mod xt_primitives;
 pub mod contract;
 pub mod balances;
 
+pub extern crate codec;
+pub extern crate log;
+
 /// Generates the extrinsic's call field for a given module and call passed as &str
 /// # Arguments
 ///
@@ -103,8 +106,8 @@ macro_rules! compose_extrinsic {
 	$call: expr
 	$(, $args: expr) *) => {
 		{
-            use codec::Compact;
-            use log::info;
+            use crate::extrinsic::codec::Compact;
+            use crate::extrinsic::log::info;
             use crate::extrinsic::xt_primitives::*;
 
             info!("Composing generic extrinsic for module {:?} and call {:?}", $module, $call);

